modules/ROOT/Edits/create-inbound-message-flow.adoc

```diff
@@ -0,0 +1,119 @@
+= Create and Configure an Inbound Message Flow
+:page-aliases: configure-message-flows.adoc
+
+You can create and configure an inbound message flow to receive EDI messages from your trading partners, including messages that go through 3rd party connections. When you create an inbound message flow, you can validate and transform the inbound messages and then send the translated message (JSON or XML) to your backend applications.
+
+To receive messages sent from a trading partner through a 3rd party connection, select an endpoint owned by that connection.
+
+=== Prerequisites
+Before you create your first inbound message flow, you must configure your xref:setup-payload-storage-API.adoc[payload storage API].
+
+=== Steps
+
+. From the Anypoint Partner Manager navigation menu, select *Message Flows*.
+. Click *New message flow > Receive from partner*.
+. Either select an existing partner from the list or select *New* to create a new partner, as described in xref:create-partner.adoc#[Create a partner].
++
+A new message flow is created.
++
+. <<receiving-endpoint,Configure the receiving endpoint>>.
+. <<source-message-type,Configure the source message type>>.
+. <<import-map,Import the translation map>>.
+. <<verify-message-flow,Verify that the message flow is complete>>.
+
+[receiving-endpoint]
+== Specify the Receiving Endpoint
+
+Specify the endpoint that will receive the message you are sending. You can do either of the following:
+
+* Select an existing endpoint owned by the host, target partner, or 3rd party connection.
++
+If you select an endpoint owned by a 3rd party connection, the inbound message goes through that connection.
++
+* Create a new receiving endpoint for this message flow.
+
+To specify the receiving endpoint:
+
+. In the message flow, expand the *Receiving from <partner>* section and click *Select endpoint*.
+. In the *Receiving endpoint* section, click *Select*.
+. In the *Select an endpoint: Receive from <partner>* window, do either of the following:
+* Select an existing receiving endpoint and click *Select*.
+* Click *New* to create a new endpoint, as described in xref:<<create-endpoint.adoc, Create and Configure Endpoints>>.
```

Comment:
Yikes, you want to review the entire doc to see how many times (and how many different wordings) this is conveyed. All this confuses the simple options of using what exists vs. creating something new.